spec/cypress/integration/CipStoriesAdmin.feature

```diff
@@ -106,4 +106,20 @@ Feature: Admin user interaction with Core Induction Programme
     Then "page body" should contain "Your changes have been saved"
     And "page body" should contain "New mentor material title"
     And "page body" should contain "New mentor material content"
-    And the page should be accessible
\ No newline at end of file
+    And the page should be accessible
+
+  Scenario: Can create content-less lessons
+    Given core induction programme with year and module
+    And I am on "core induction programme year" page with id "a4dc302c-ab71-4d7b-a10a-3116a778e8d5"
+
+    When I click on "link" containing "Create CIP Lesson"
+    Then "page heading" should contain "Create lesson"
```

Comment:
Let's use 
```
Then I should be on "core induction programme mentor material edit" page
```
step if what we want to check here is the location.

Also, let's add 
```
And the page should be accessible
```

Reply:
done